pkg/pool-weighted/contracts/managed/ManagedPoolSettings.sol

```diff
@@ -94,6 +99,7 @@ abstract contract ManagedPoolSettings is BasePool, ProtocolFeeCache, ReentrancyG
     event AllowlistAddressRemoved(address indexed member);
     event TokenAdded(IERC20 indexed token, uint256 normalizedWeight);
     event TokenRemoved(IERC20 indexed token);
+    event CircuitBreakerSet(IERC20 indexed token, uint256 lowerBoundPercentage, uint256 upperBoundPercentage);
```

Comment:
```suggestion
    event CircuitBreakerSet(IERC20 indexed token, uint256 bptPrice, uint256 lowerBoundPercentage, uint256 upperBoundPercentage);
```
I think we'd need to value to be able to make the necessary calculations offchain.